Dockerfile

```diff
@@ -32,13 +33,13 @@ RUN apt-get update && \
         libgdal-dev  \
         gcc \
         g++ \
-        python3.7-dev && \
+        python3.8-dev && \
```

Comment:
You should check if `python3-dev` is available and working. IMO it's better to avoid specific python versions whenever possible .